api/src/neo4j/queryHandlers/single.js

```diff
@@ -8,7 +8,17 @@ const querySingleResult = async (statement) => {
 
   try {
     const response = await session.run(statement);
+
+    if (response.records.length === 0) {
+      throw new Error('404');
+    }
+
     result = response.records[0].get(0);
+
+    if (Object.values(result).flat().filter(x => x !== 0).length === 0) {
```

Comment:
Should we instead fail this request at source instead of allowing a broken response?

Reply:
What do you mean by "at source"? In the cypher query?

Reply:
Yes. Maybe we can discuss about this today, let's not drag out this PR any longer.

Reply:
For reference, there is a new card MA-932.